pkgs/development/libraries/libpfm/default.nix

```
@@ -9,7 +9,7 @@ stdenv.mkDerivation rec {
     sha256 = "0jabhjx77yppr7x38bkfww6n2a480gj62rw0qp7prhdmg19mf766";
   };
 
-  installFlags = "DESTDIR=\${out} PREFIX= LDCONFIG=true";
+  makeFlags = [ "DESTDIR=\${out}" "PREFIX=" "LDCONFIG=true" "ARCH=${stdenv.hostPlatform.parsed.cpu.name}" ];
```

It does not seem to make any difference at the moment but if the library ever stops being relocatable, we should specify a proper prefix.

```suggestion
  makeFlags = [ "PREFIX=${placeholder "out"}" "LDCONFIG=true" "ARCH=${stdenv.hostPlatform.parsed.cpu.name}" ];
```

---

Done